great_tables/_text.py

```diff
@@ -45,24 +52,23 @@ def _md_html(x: str) -> str:
 
 
 def _process_text(x: str | Text | None) -> str:
+    from great_tables._helpers import UnitStr
```

Comment:
needs tests. If UnitStr isn't a subclass of text it needs to be added to the type hint on line 54

Reply:
A number of tests for `UnitStr` were added during pairing.